src/wallet/wallet2.cpp

```diff
@@ -3424,6 +3447,44 @@ bool wallet2::load_keys(const std::string& keys_file_name, const epee::wipeable_
         m_device_name = m_key_device_type == hw::device::device_type::LEDGER ? "Ledger" : "default";
       }
     }
+    
+    if (json.HasMember("original_keys_available"))
+    {
+      GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, original_keys_available, int, Int, false, false);
+      m_original_keys_available = field_original_keys_available;
+      if (m_original_keys_available)
+      {
+	GET_FIELD_FROM_JSON_RETURN_ON_ERROR(json, original_address, std::string, String, false, std::string());
+	if (!field_original_address_found)
+	{
+	  LOG_ERROR("Field original_address not found in JSON");
+	  return false;
+	}
+	address_parse_info info;
+	bool ok = get_account_address_from_str(info, m_nettype, field_original_address);
+        if (!ok)
+	{
```

Comment:
A bit borked here too

Oh I get it, I looked in vi and you added tabs that's why it's fucked :/
I hate how github hides them...

Reply:
Yes, it's only tabs. I am innocent, my IDE did it :) Replaced with blanks now.